config/endpoints/2_dgfip_attestation_fiscale.yml

```diff
@@ -4,35 +4,43 @@
   path: '/v3/dgfip/attestations_fiscales/{siren}'
   perimeter:
     description: "
-    ⚠️ Ce endpoint ne concerne que les entreprises n'appartenant pas à un groupe de sociétés
-  imposé selon le régime fiscale d'intégration (article 223 A du CGI), dit groupe IS ;
-  ou à un groupe de sociétés ayant opté pour la consolidation du paiement de la TVA
-  (article 1693 ter du CGI), dit groupe TVA : pour ces cas, veuillez vous référer
-    aux endpoints relatifs (pas encore disponibles).
-      \n
-      \n
+    
+    **Entreprises concernées :**
+    \n
+    \n
+    Cette API ne concerne pas **❌ les sociétés "filles" appartenant à un groupe de société**.
```

Comment:
Tu as vérifié que `les sociétés "filles" appartenant à un groupe de société` correspond **exactement** aux groupes IS (au sens de l'article 223 A du CGI) **et** aux groupes TVA (au sens de l'article 1693 ter du CGI) ? Parce que sinon cette phrase est fausse.

Ici pour moi il faut être plus précis: on parle à des gens qui sont censé connaître un minimum la législation.

Reply:
imo tu peux faire un mix entre cette phrase et l'ancienne phrase de ce style:

```
Cette API ne concerne pas **❌ les sociétés "filles" appartenant à un groupe de sociétés** défini comme:
* un groupe de sociétés imposé selon le régime fiscale d'intégration (article 223 A du CGI), dit groupe IS ;
* groupe de sociétés ayant opté pour la consolidation du paiement de la TVA (article 1693 ter du CGI), dit groupe TVA

Pour ces 2 cas merci de vous référer aux endpoints FEEDME pas encore dispo
```

Reply:
@skelz0r Je préfère garder la formulation ci-dessus, qui est validée par le document transmis dans le dernier commit. La DGFIP elle-même utilise des formulations plus simples pour expliquer le service à ses usagers.